lms/envs/production.py

```diff
@@ -943,6 +943,12 @@ def get_env_setting(setting):
 ########################## limiting dashboard courses ######################
 DASHBOARD_COURSE_LIMIT = ENV_TOKENS.get('DASHBOARD_COURSE_LIMIT', None)
 
+######################## Setting for django-countries ########################
+# django-countries provides an option to make the desired countries come up in
+# selection forms, if left empty countries will come up in ascending order as before
+# https://github.com/SmileyChris/django-countries#show-certain-countries-first
+COUNTRIES_FIRST = []
```

Comment:
This should read from the `env` tokens.

```suggestion
COUNTRIES_FIRST = ENV_TOKENS.get('DJANGO_COUNTRIES_COUNTRIES_FIRST', [])

```

Reply:
@0x29a Can you handle testing/reviewing this change after it's implemented?

Reply:
Not sure if this the best name for the variable.

Reply:
@giovannicimolin, yep, I'll test this again.